test/Writer/Renderer/Entry/AtomTest.php

```diff
@@ -62,7 +62,8 @@ public function tearDown()
     public function testRenderMethodRunsMinimalWriterContainerProperlyBeforeICheckAtomCompliance()
     {
         $renderer = new Renderer\Feed\Atom($this->validWriter);
-        $renderer->render();
+
+        $this->assertInstanceOf(Renderer\Feed\Atom::class, $renderer->render());
```

Comment:
I am not sure about this test, we tend to drop fluent interfaces, so we would need to change it back when we do it.
Or then we will use `assertNull`?

I know, that this is reported as "risky" because of no assertion.

Reply:
When the lack of exception is the assertion, `$this->addToAssertionCount(1);` is a good choice instead of meaningless or unrelated assertion.

Reply:
The entire API is not good and it will change with the next major version but that is not the goal of this pull request..

Reply:
Tangential: This test does not assert what the name states imo. Test name needs to be amended or test updated.

Reply:
I do not agree that `addToAssertionCount` is better in that case. This does not give us any information at all.
I've checked it shortly and people wanted something like "assertNotThrowException" but it never happened in PHPUnit. Assertion on the result of the method is the best imo. If we drop fluent interfaces (in next major) we will change it to `assertNull` and all will be fine. At least for now there is no distinction in null and void in PHP ;-)

Reply:
I strongly disagree. Return type is beside the point for the test. If it changes it would not change the validity of the test outcome. More than that, asserting return value in any form actually distracts from the actual implicit test condition. It is better to show that we have no explicit assertions and extra attention should be paid to the test.

Reply:
I think we are looking for [`@doesNotPerformAssertions`](https://phpunit.readthedocs.io/en/8.5/annotations.html#doesnotperformassertions) here.

But personally, as I said, I would go with assertion of the result. I know it tests two things, and we have "single assertion tests" but it's better that tricking PHPUnit that we have assertion, once we haven't.

Reply:
@michalbundyra That annotation was first implemented in 5.6.0. I guess I can start using it already :joy: